.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 .idea/
 .vscode/
+Cargo.lock
```

Comment:
I think today its recommended to commit the Cargo.lock, so we should probably do that if we don't do so yet.